features/support/hooks.rb

```diff
@@ -39,6 +39,15 @@
   end
 end
 
+Before '@narrow-screen' do
+  @browser_width, @browser_height = page.driver.browser.manage.window.size.to_a
```

Comment:
btw ideally we should set these values as constants somewhere. But anyway, not super important.